src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -509,7 +509,7 @@ public boolean isMeasureByDefault()
         ColumnType Barcode = new ColumnTypeImpl("Unique ID", "string", "http://www.labkey.org/types#storageUniqueId", null);
         ColumnType TextChoice = new ColumnTypeImpl("Text Choice", "string", "http://www.labkey.org/types#textChoice", null);
         ColumnType SMILES = new ColumnTypeImpl("SMILES", "string", "http://www.labkey.org/exp/xml#smiles", null);
-
+        ColumnType Calculation = new ColumnTypeImpl("Calculation", "http://www.labkey.org/exp/xml#calculated");
```

Comment:
"`http://www.labkey.org/exp/xml#calculated`" is the `conceptURI` for this type, not the `rangeURI`. I think we want something like this:
```suggestion
        ColumnType Calculation = new ColumnTypeImpl("Calculation", null, "http://www.labkey.org/exp/xml#calculated", null);
```
Though it doesn't really matter unless we add API support for calculated columns

Reply:
Changed.
The plan is to add API support for calculated columns. I have to open a bug and assign it to Cory.